db/models.py

```diff
@@ -1,8 +1,15 @@
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
 
+import settings
```

Comment:
it's better to use
`from django.conf import settings` because you may want to have settings folder and python will import it instead of real settings